### PR TITLE
data: ralphsulit

### PR DESCRIPTION
**Pull request by ralphsulit**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8601"></a>

### CODE REVIEW

File: data/ralphsulit.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/ralph-sulit/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/ralphsulit",
+      "icon": "FaGithub"
+    },
```
Comment (sital002) [inline data/ralphsulit.json:20]:
```suggestion
    }
```
This is a trailing comma which isnot required.  Please remove it.

**Review by sital002 — CHANGES_REQUESTED** [review]

File: data/ralphsulit.json (hunk 1 above)
Comment (sandramsc) [inline data/ralphsulit.json:20]: Hi, consider removing this comma (**,**) on line 20, otherwise it renders th efile invalid.
```suggestion
    }
```

**Review by sandramsc — CHANGES_REQUESTED** [review]

**Review by accodes21 — CHANGES_REQUESTED** [review]
No need for ```avatar``` kindly remove it.

**Comment by ralphsulit** [comment]
removed avatar and trailing comma. thank you.

**Review by sital002 — APPROVED** [review]
Looks good 👍

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**